src/Tree/Entity/Repository/ClosureTreeRepository.php

```diff
@@ -150,10 +150,21 @@ public function childrenQueryBuilder($node = null, $direct = false, $sortByField
         }
 
         if ($sortByField) {
-            if ($meta->hasField($sortByField) && in_array(strtolower($direction), ['asc', 'desc'], true)) {
-                $qb->orderBy('node.'.$sortByField, $direction);
+            if (is_array($sortByField)) {
+                foreach ($sortByField as $key => $field) {
+                    $fieldDirection = strtolower(is_array($direction) ? ($direction[$key] ?? 'asc') : $direction);
+                    if ($meta->hasField($field) && in_array($fieldDirection, ['asc', 'desc'], true)) {
```

Comment:
This breaks my proejct as the `root` is a reference and not a field. The error I get is `Invalid sort options specified: field - root, direction - ASC`

Reply:
@pmishev can you please take a look at this? (also https://github.com/doctrine-extensions/DoctrineExtensions/issues/2492)